### PR TITLE
Enable ssl on ebl

### PR DESCRIPTION
**Pull request by ptolemy428**
## Approvals

- [x] Verified
- [x] @TimMoore
- [ ] @hebbo
- [x] @ptolemy428
- [ ] @iiman

### CODE REVIEW

File: bin/format [hunk 1]
```diff
@@ -5,6 +5,6 @@ set -eu
 
 script_dir=$(dirname "$0")
 
-export DOCKER_CWD=$PWD/cloudformationcore
+export DOCKER_CWD=$PWD
```
Comment (ptolemy428) [inline bin/format]: Fix bug that causes tests not being formatted by yapf.

File: cloudformationcore/load_balancer.py [hunk 2]
```diff
@@ -13,9 +13,10 @@ def create_internal_load_balancer(custom_resource, subnet_keys,
             Timeout=300, ),
         Listeners=[
             elb.Listener(
-                LoadBalancerPort="80",
```
Comment (ptolemy428) [inline cloudformationcore/load_balancer.py]: No more http, only https going forward. encrypting everything!

**Comment by ptolemy428** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;saUMSkdI&#x2F;470-ssl-for-murmur-event-hub-communication"> SSL for murmur --&gt; event_hub communication</a>

**Comment by TimMoore** [comment]
Looks good

**Comment by ptolemy428** [comment]
@TimMoore. that was quick! :)

**Pull request merged**